keyboards/kprepublic/bm80/keymaps/peepeetee/config.h

```diff
@@ -0,0 +1,132 @@
+// /* Copyright 2021 peepeetee
```

Comment:
This file should be deleted, as it's not being used, at all

Reply:
I'm currently testing out the default config but with my own keymap, I'll uncomment config.h once I'm satisfied with the default LED config

Reply:
Well, you can use git to ignore changes to this file, if needed.  May make things easier, long term, too.

Reply:
Re-enabled